v21.2/changefeed-sinks.md

```diff
@@ -130,6 +130,14 @@ Examples of supported cloud storage sink URIs:
 'gs://{BUCKET NAME}/{PATH}?AUTH=specified&CREDENTIALS={ENCODED KEY}'
 ~~~
 
+### HTTP
+
+~~~
+http://localhost:8080/{PATH}
```

Comment:
Question: does the HTTP URI need to be single-quoted like all other sink targets URIs above it?